### PR TITLE
Loudness/RIAA Curve

### PR DESCRIPTION
**Pull request by privatezero**
Adds some normalization and equalization info as well as resolving most of https://github.com/amiaopensource/ffmprovisr/issues/178.

Hope to get some more in soon (such as standard normalization and dynaudnorm) but this is a start!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1081,6 +1081,111 @@
   <!-- ends append notice to access mp3 -->
 
   </div>
+
+  <div class="well">
+  <h4>Normalize/Equalize Audio</h4>
+  <!-- loudnorm metadata -->
```
Comment (retokromer) [inline index.html]: The comment should be moved at the beginning of the recipe.

File: index.html [hunk 2]
```diff
@@ -1081,6 +1081,111 @@
...
+        <div class="well">
+          <h3>Calculate Loudness Levels</h3>
+          <p><code>ffmpeg -i <i>input_file</i> -af loudnorm=print_format=json -f null -</code></p>
+          <p>This filter calculates and outputs loudness information in json about an input file (labeled input) as well as what the levels would be if loudnorm were applied in its one pass mode (labeled output). The values generated can be used as inputs for a 'second pass' of the loudnorm filter allowing more accurate loudness normalization than if it is used in a single pass.</p>
+          <p>These instructions use the loudnorm defaults, which align well with PBS recommendations for target loudness. More information can be found at the <a href=https://ffmpeg.org/ffmpeg-filters.html#loudnorm>loudnorm documentation</a>.</p>
+                    <p>Information about PBS loudness standards can be found in the <a href=http://www-tc.pbs.org/capt/Producing/TOS-2012-Pt2-Distribution.pdf>PBS Technical Operating Specifications</a> document. Information about EBU loudness standards can be found in the <a href=https://tech.ebu.ch/docs/r/r128-2014.pdf>EBU R 128</a> recommendation document.</p>
```
Comment (retokromer) [inline index.html]: Please reduce indent.

File: index.html [hunk 3]
```diff
@@ -1081,6 +1081,111 @@
...
+      <div class="modal-content">
+        <div class="well">
+          <h3>RIAA Equalization</h3>
+          <p><code>ffmpeg -i <i>input_file</i> -af aemphasis=type=riaa <i>output_file</i></code></p>
+          <p>This will apply RIAA equalization to an input file allowing correct listening of audio transferred 'flat' (without EQ) from records that used this EQ curve. For more information about RIAA equalization see the <a href=https://en.wikipedia.org/wiki/RIAA_equalization>Wikipedia page</a> on the subject.</p>
+          <dl>
+            <dt>ffmpeg</dt><dd>starts the command</dd>
+            <dt><i>input_file</i></dt><dd>path, name and extension of the input file</dd>
+            <dt>-af aemphasis=type=riaa</dt><dd>activates the aemphasis filter and sets it to use RIAA equalization</dd>
+            <dt><i><i>output_file</i></i></dt><dd>path and name of output file</dd>
```
Comment (retokromer) [inline index.html]: Please remove the double `<i>` and `</i>`.

File: index.html [hunk 4]
```diff
@@ -1081,6 +1081,111 @@
...
+          <p>Information about PBS loudness standards can be found in the <a href=http://www-tc.pbs.org/capt/Producing/TOS-2012-Pt2-Distribution.pdf>PBS Technical Operating Specifications</a> document. Information about EBU loudness standards can be found in the <a href=https://tech.ebu.ch/docs/r/r128-2014.pdf>EBU R 128</a> recommendation document.</p>
+          <dl>
+            <dt>ffmpeg</dt><dd>starts the command</dd>
+            <dt><i>input_file</i></dt><dd>path, name and extension of the input file</dd>
+            <dt>-af loudnorm</dt><dd>activates the loudnorm filter with default settings</dd>
+            <dt>dual_mono=true</dt><dd>(optional) Use this for mono files meant to be played back on stereo systems for correct loudness. Not necessary for multi-track inputs.</dd>
+            <dt><i>output_file</i></dt><dd>path, name and extension for input file</dd>
```
Comment (retokromer) [inline index.html]: **_output_** file

File: index.html [hunk 5]
```diff
@@ -1081,6 +1081,111 @@
...
+            <dt>-af loudnorm</dt><dd>activates the loudnorm filter with default settings</dd>
+            <dt>dual_mono=true</dt><dd>(optional) use this for mono files meant to be played back on stereo systems for correct loudness. Not necessary for multi-track inputs.</dd>
+            <dt>measured_I=<i>input_i</i></dt><dd>use the 'input_i' value (integrated loudness) from the first pass in place of input_i</dd>
+            <dt>measured_TP=<i>input_tp</i></dt><dd>use the 'input_tp' value (true peak) from the first pass in place of input_tp</dd>
+            <dt>measured_LRA=<i>input_lra</i></dt><dd>use the 'input_lra' value (loudness range) from the first pass in place of input_lra</dd>
+            <dt>measured_LRA=<i>input_thresh</i></dt><dd>use the 'input_thresh' value (threshold) from the first pass in place of input_thresh</dd>
+            <dt>offset=<i>target_offset</i></dt><dd>use the 'target_offset' value (offset) from the first pass in place of target_offset</dd>
+            <dt>linear=true</i></dt><dd>tells loudnorm to use linear normalization</dd>
+            <dt><i>output_file</i></dt><dd>path, name and extension for input file</dd>
```
Comment (retokromer) [inline index.html]: **_output_** file

Comment (privatezero) [reply index.html]: eeek! That's what I get for being lazy and cut/pasting!

**Review by retokromer — APPROVED** [review]

**Pull request merged**